react/fields/Dropdown/Dropdown.js

```diff
@@ -178,7 +137,7 @@ export default class Dropdown extends Component {
 
     return (
       <div className={classNames}>
-        {this.renderLabel()}
+        <FieldLabel {...this.props} />
```

Comment:
If you explicitly grab the props required for FieldLabel you have to have the matching propType validation in every single component that is using FieldLabel :(